sprocs/grading_jobs_insert_internal.sql

```diff
@@ -25,15 +25,17 @@ DECLARE
     variant_id bigint;
     instance_question_id bigint;
     assessment_instance_id bigint;
-    grading_method enum_grading_method;
+    grading_method_internal boolean;
+    grading_method_external boolean;
+    grading_method_manual boolean;
     new_correct boolean;
 BEGIN
     -- ######################################################################
     -- get the related objects
 
     -- we must have a variant, but we might not have an assessment_instance
-    SELECT s.credit,       v.id, q.grading_method,              iq.id,                  ai.id
-    INTO     credit, variant_id,   grading_method, instance_question_id, assessment_instance_id
+    SELECT s.credit,       v.id, q.grading_method_internal, q.grading_method_external, q.grading_method_manual,                iq.id,                  ai.id
```

Comment:
It does not necessarily make sense to insert the grading method booleans from the question into the grading_job table. 

Perhaps we should define an internal, external, and manual grading operation and once they complete, update the grading job. Or we can still delegate the calls and include a flag to update the boolean based on which delegated call was hit.

This is my first pass at this logic, so I still have to digest and think about this. Some input from @mwest1066  would be quite valuable here too.